tests/lib/snaps/test-snapd-mount-control/bin/mntctl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```

Comment:
is there a reason we can't just use `/bin/mount` instead of this Python?

Reply:
Honestly, I don't remember :-)

One reason might be that we'd need to widen the AppArmor rules a bit, because these two rules are also needed:
```
  /{,usr/}bin/mount ixr,
  /run/mount/utab* wrlk,
```
(interestingly, it looks like permissions to run `umount` are not needed, yet we are calling it in these spread test). If you don't see a problem with adding this rules (the first one is certainly safe, the second one I'm not 100% sure about), then I can certainly drop the python script.

Reply:
Yeah, providing at least /bin/mount via the mount-control interface seems perfectly logical to me, and as per what we just discussed on IRC, the /run/mount/utab* rule is also fine too